pallets/asset/src/tests.rs

```diff
@@ -2839,3 +2839,69 @@ fn asset_vc_status_change_with_wrong_asset_id_should_fail() {
 		);
 	});
 }
+
+/// Test case for validating that asset creation fails when provided with an invalid asset value or out of space.
+fn asset_create_should_fail_invalid_asset_value() {
```

Comment:
nit: `asset_create_should_fail_for_invalid_asset_value()`

Reply:
@vatsa287 thanks for the review. I will fix the naming issue and also add the `assert_qty` assertion error in next PRs. thank you.